service/job.go

```diff
@@ -101,7 +104,10 @@ func (s *CaptionsService) GetJob(r *http.Request) (int, interface{}, error) {
 	// TODO: on the 3play client, we should look at the errors field and check for not_found errors at least
 	job, err := s.client.GetJob(id)
 	if err != nil {
-		return http.StatusNotFound, nil, captionsError{err.Error()}
+		if err == database.ErrJobNotFound {
```

Comment:
This comparison will work correctly only if the `s.client. GetJob` doesn't wrap `ErrJobNotFound`. Is this something we can take for granted?

Reply:
I think that's alright. We have a test in place that would  break in case it wraps.

Reply:
👌